conf/systemd.service

```
@@ -9,7 +9,7 @@ Type=simple
 User=__APP__
 Group=__APP__
 WorkingDirectory=__INSTALL_DIR__/
-ExecStart=__INSTALL_DIR__/node_exporter
+ExecStart=__INSTALL_DIR__/node_exporter --web.listen-address 127.0.0.1:9100 --web.listen-address __IP_YUNOHOST_SERVER__:__PORT__
```

```suggestion
ExecStart=__INSTALL_DIR__/node_exporter --web.listen-address __BIND_ADDRESS__:__PORT__
```

et définir `$bind_address="127.0.0.1"` si external = false
     ou bien `$bind_address="0.0.0.0"` si external = true